public-images/public-images-how-to/use-local-cloud-init-ds.rst

```diff
@@ -0,0 +1,96 @@
+.. _use-local-ds:
+
+Create and Use a Local Cloud-Init Datasource
+============================================
+
+Purpose
+-------
+
+The steps in this How-To guide will allow you to create a seed image that defines a custom cloud-init datasource.
```

Comment:
```suggestion
The steps in this how-to guide will allow you to create an Ubuntu seed image that defines a custom cloud-init datasource.
```
Does the above suggestion make sense? When you say seed image, I usually assume something like a core Ubuntu image that can be used as the starting point for adding further packages or making changes that customise it.